### ISSUE TITLE
Config Export messes up last line

### ISSUE THREAD

**Issue by jonpas** [acemod/ACE3#1158]
ACE3 Version: 3.0.0.3 cf635d45645f5813f308ff2d764ff67d4d30b182

**Mods:**
* `@cba_a3`
* `@ace3`

**Placed ACE3 Modules:**
* Allow Config Export

**Description:**
Config Export messes up last line in the pasted export, examples:
```
class ace_medical_medicSetting_SurgicalKit {
    value = 1;
    typeName = "SCALAR";
;
```
```
class ace_advanced_ballistics_simulationRadius {
    value = 3000;
    typeName = "SCALAR";
    force = 1;
};;
```

**Steps to reproduce:**
* Place `Allow Config Export` module
* Export Config
* Check last line in pasted export

**Where did the issue occur?**
Singplayer - Editor

**RPT log file:**
N/A

Issue found by @Jastreb

**Comment by Jastreb** [comment]
Confirmed, same with PWS final version of ACE3. Making server crash on start:

![Server crash on start](http://prntscr.com/74vxoe)

Nothing in RPT.

**Comment by jaynus** [comment]
Is this because of my clipboard exports or because of the builder, @glowbal?

**Comment by Jastreb** [comment]
Still issue in 3.0.1

**Comment by PabstMirror** [comment]
`CBA_fnc_split` doesn't work past 10,000 chars in non-scheduled.  I think we also stick an extra `;` on the end of any output.

We could just spawn that part of the code or:

we could chunk data to the extension via:
```
_countSent = 0;
while {_countSent < (count _compiledConfig)} do {
    "ace_clipboard" callExtension (_compiledConfig select [_countSent, CHARS_PER_EXT_CALL]);
    _countSent = _countSent + CHARS_PER_EXT_CALL;
};
"ace_clipboard" callExtension "--COMPLETE--";
```

but I've been getting occasional crashes with both it and the original version.

**Comment by Glowbal** [comment]
Why do we need to split the input?

**Comment by PabstMirror** [comment]
Biki says `There is a maximum size for how much data an extension can transfer to the game. When the feature was initially released it was 4k, it has since been raised to 8k, 16k, and is now down to 10kB. So far it has only changed between versions of arma, not while the particular arma process is running.`

Although the line above it mentions java, so the data might be a little old :smile:  and it's talking about return data length, but I'm guessing there is some kind of max string length for input.

I tried sending 9000 chars at once.  I only got 8191 in my clipboard and my game crashed.
